root/usr/share/nethesis/NethServer/Module/Dashboard/SystemStatus/Backup.php

```diff
@@ -48,7 +48,45 @@ private function readBackup()
         $backup['vfs'] = $br['VFSType'] ? $br['VFSType'] : '-';
         $backup['status'] = $br['status'];
         $backup['time'] = $br['BackupTime'];
-
+        
+        $orario = explode(" ",$backup['time']);
+      
+        function formatOre($h){
+	        if(strlen($h)==1){return "0".$h;}else{return $h;}
```

Comment:
Please respect the general indentation rules: we are not using a strict standard but please write it a readable manner.
Take a look at: https://www.php-fig.org/psr/psr-2/

In this case you could also use a ternary operator or much better http://php.net/manual/en/function.sprintf.php